tools/olm-bundle/README.md

```diff
@@ -0,0 +1,17 @@
+# OLM bundle generator
```

Comment:
I guess the README should describe things which need to be updated manually? Following come to my mind:
* Description inside the CSV
* `permissions` and `clusterPermissions` inside the CSV (or is this in some way generated)?
* Examples in the CSV annotation?

Reply:
Just added a section in the README that describes what is generated automatically and what needs to be updated manually